### DISCUSSION TITLE
Export as geopdf for use with Avenza (or other iphone mapping software), symbols are wrong

### DISCUSSION THREAD

**Discussion by markaoconnell** [OpenOrienteering/mapper#1956] [Questions & Answers]
I'm using OpenOrienteeringMapper 0.9.4 on a Windows box and am having trouble creating a geo-referenced pdf (for use with the Avenza app on an iphone).

I use the File -> Export option, Geospatial vector data, then name file xxx.pdf - it seems to export fine, and linear features seem to show fine (contours, trails), but most symbols just show up as X's (boulders, depressions, contour slope lines, etc) while filled in features (e.g. vegetation) isn't transparent and it hides the contours and other features underneath.

[Sandra Pond Avenza.pdf](https://github.com/OpenOrienteering/mapper/files/6488378/Sandra.Pond.Avenza.pdf)

I've attached an example of the output I get - it seems likely that I am making an elementary mistake, but I can't figure out what!  Any help would be appreciated.

**Comment by dg0yt** [comment]
Thank you very much for the feedback. 

You are not making a mistake. The geospatial vector export (since v0.9.2, March 2020) is entirely based on GDAL. It came nearly for free, but most of these drivers have some limitations:
- Some symbol details cannot be expressed in GDAL feature style language,
- Some valid some feature style language expression are not handled by particular drivers.
In particular, I didn't find an easy way to combine both the existing graphical PDF export with the geospatial one. 

There is an open ticket for GeospatialPDF #795 which you might subscribe to.

**Reply by dg0yt** [reply]
PS: Basically, with the geospatial vector data export in its current form, the focus is on preserving geospatial attributes, not on preserving feature style.

**Comment by dg0yt** [comment]
For Advenza, GeoTIFFs might be an alternative. However, they are raster data, and Mapper doesn't create tiles or hierarchy for limiting memory demand in the target app.

**Reply by markaoconnell** [reply]
Thanks for the answers, that was helpful and nice to know that it wasn’t some simple mistake on my part.
I tried a GeoTiff, but when I did the export and put either a .tiff or .tif extension, I got an error that there was no driver to produce that file.  When I look here: https://www.openorienteering.org/mapper-manual/pages/gdal.html I see tiff as an import format but not an export format.  Let me look and see if I can convert one of the export formats to tiff, perhaps that would work?

**Reply by dg0yt** [reply]
Again, probably no fault on your side, but a missing feature in Mapper. The raster graphics export doesn't use gdal. You might try KMZ.

**Reply by hedger48** [reply]
> Thanks for the answers, that was helpful and nice to know that it wasn’t some simple mistake on my part. I tried a GeoTiff, but when I did the export and put either a .tiff or .tif extension, I got an error that there was no driver to produce that file. When I look here: https://www.openorienteering.org/mapper-manual/pages/gdal.html I see tiff as an import format but not an export format. Let me look and see if I can convert one of the export formats to tiff, perhaps that would work?

Hello, did you find a way to make this work?  I also look for way to put geo red pot or tiff on iPhone. Thx

**Comment by markaoconnell** [comment]
No I gave up eventually…

Mark
